### DISCUSSION TITLE
where can i buy coffee

### DISCUSSION THREAD

**Discussion by aamfahrur** [salman0ansari/whatsapp-api-nodejs#467] [Announcements]
where can i buy coffee for the author of this project?

**Comment by salman0ansari** [comment]
> where can i buy coffee for the author of this project?

https://www.buymeacoffee.com/salman0ansari

**Comment by luispimenta** [comment]
@salman0ansari enable that for you github account 

https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account

**Comment by salman0ansari** [comment]
> @salman0ansari enable that for you github account
> 
> https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account

submitted for approval! 🤞🏻

**Comment by salman0ansari** [comment]
Approved 🥳

**Comment by luispimenta** [comment]
Works! 🚀🎉

**Comment by salman0ansari** [comment]
> Works! rockettada

Thank You!

**Comment by sreekanthmr** [comment]
Hey @salman0ansari great work.  Unfortunately, payments from India are getting rejected.  I'll try alternative sources though.

**Comment by salman0ansari** [comment]
> Hey @salman0ansari great work. Unfortunately, payments from India are getting rejected. I'll try alternative sources though.

I am from India,
You can do UPI
connect with me here
https://telegram.dog/salman0ansari
